Dockerfile

```diff
@@ -1,71 +1,40 @@
-FROM erlang:22.3.2-alpine as node-builder
-RUN apk add --no-cache --update \
-    git tar build-base linux-headers autoconf automake libtool pkgconfig \
-    dbus-dev bzip2 bison flex gmp-dev cmake lz4 libsodium-dev openssl-dev \
-    sed wget curl
+ARG UBUNTU_BUILDER=ubuntu:18.04
+ARG UBUNTU_RUNNER=quay.io/team-helium/blockchain-node:2021-09-24-ubuntu18
 
-ENV CC=gcc CXX=g++ CFLAGS="-U__sun__" \
-    ERLANG_ROCKSDB_OPTS="-DWITH_BUNDLE_SNAPPY=ON -DWITH_BUNDLE_LZ4=ON" \
-    ERL_COMPILER_OPTIONS="[deterministic]" \
-    PATH="/root/.cargo/bin:$PATH" \
-    RUSTFLAGS="-C target-feature=-crt-static"
+FROM $UBUNTU_BUILDER as rosetta-builder
 
-# install Rust toolchain
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+COPY . /src/rosetta
```

Comment:
We can't copy from relative dir. We need this docker file to be able to run on its own due to deploy env reasons, that's why I had the git clone line.